README.rst

```diff
@@ -21,6 +21,8 @@ Please use the `main ZMON repository`_ to start a Vagrant demo box.
 
 Make sure the provided Vagrant-Box is up and all services are running.
 
+Update your GitHub ``client-id`` and ``client-secret`` in ``zmon-controller-app/src/main/config/application-github.yml``.
```

Comment:
as already mentioned: I would not suggest this to users (only leads to people changing files in git and accidentally committing them --- better suggest using Java System Properties or environment variables